python/ql/src/Security/CWE-730/PolynomialBackTracking.ql

```diff
@@ -0,0 +1,6 @@
+import python
+import semmle.python.regex.SuperlinearBackTracking
+
+from PolynomialBackTrackingTerm t
+where t.getLocation().getFile().getBaseName() = "KnownCVEs.py"
+select t.getRegex(), t, t.getReason()
```

Comment:
This one looks more like a file that should be under `test/` and not `src/`?

Reply:
It does, yes..